apps/frontend/src/lib/message/edge.ts

```diff
@@ -0,0 +1,49 @@
+import { z } from "zod";
+import {
+  MessageTypeSchema,
+  MessageType
+} from "@types";
+
+import { EdgeMessageContentSchema } from "@/lib/storage/types";
+
+export const EdgeMessageSchema = z.object({
+  type: MessageTypeSchema,
+  edge: EdgeMessageContentSchema,
+});
+
+export type EdgeMessage = z.infer<typeof EdgeMessageSchema>;
+
+export interface GenerateEdgeMessageArgs {
+  edgeId: string,
+  senderUsername: string,
```

Comment:
what is the role of `senderUsername` in this message?

Reply:
The other user needs your name to set in their local edge info.